src/Development/IDE/GHC/Compat.hs

```diff
@@ -398,3 +399,13 @@ getConArgs = GHC.getConDetails
 
 getPackageName :: DynFlags -> Module.InstalledUnitId -> Maybe PackageName
 getPackageName dfs i = packageName <$> lookupPackage dfs (Module.DefiniteUnitId (Module.DefUnitId i))
+
+disableWarningsAsErrors :: DynFlags -> DynFlags
+disableWarningsAsErrors df =
+    flip gopt_unset Opt_WarnIsError $ foldl' wopt_unset_fatal df [toEnum 0 ..]
```

Comment:
Does `minBound` work here? Or even better, `enumerate` from Data.List.Extra?

Reply:
I copied this code from the source (GHC) as the function isn't exported in older GHC versions